autoload/pdv.vim

```diff
@@ -146,7 +152,20 @@ func! s:ApplyIndent(text, indent)
 endfunc
 
 func! pdv#ParseClassData(line)
+	let l:i = 1
+	let l:curlys = []
 	let l:text = getline(a:line)
+	let l:curlys = matchlist(l:text, '^.*\({\).*$')
+	while l:i <= 10
```

Comment:
I don't get what this loop actually does. Could you add a comment or even better move it to a function?